### PR TITLE
Updated gemspec and gemfile for v2.0.0 gem release

### PR DESCRIPTION
**Pull request by hossenlopp**
Updates the gemspec to be 2.0.0 and uses all new updated and published gems instead of git commit references for our gems. 

Pull requests into Bonnie Bundler require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-1198
- [x] JIRA ticket links to this PR
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced. N/A
- [x] Tests are included and test edge cases. N/A only gemfile updates
- [x] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [x] Code coverage has not gone down and all code touched or added is covered.
    * In rare situations, this may not be possible or applicable to a PR. In those situations:
        1. Note why this could not be done or is not applicable here:
        2. Add TODOs in the code noting that it requires a test
        3. Add a JIRA task to add the test and link it here:
- [x] Automated regression test(s) pass

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA test links: N/A
- [x] Justification for using JIRA tests: N/A
- [x] JIRA tests have been added to sprint  N/A

**Reviewer 1:**

Name: @edeyoung
- [X] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [X] JIRA tests have been run and pass N/A
- [X] You agree with the justification for use of JIRA tests or have provided input on why you disagree N/A

**Reviewer 2:**

Name: @ssayer
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -190,48 +185,52 @@ GEM
...
-    sprockets-rails (3.2.0)
-      actionpack (>= 4.0)
-      activesupport (>= 4.0)
-      sprockets (>= 3.0.0)
+    sprockets (2.12.4)
+      hike (~> 1.2)
+      multi_json (~> 1.0)
+      rack (~> 1.0)
+      tilt (~> 1.1, != 1.3.0)
+    sprockets-rails (2.3.3)
```
Comment (edeyoung) [inline Gemfile.lock:222]: this went down a version. is this expected?

File: Gemfile.lock [hunk 2]
```diff
@@ -190,48 +185,52 @@ GEM
...
-    therubyracer (0.12.1)
-      libv8 (~> 3.16.14.0)
+    therubyracer (0.12.3)
+      libv8 (~> 3.16.14.15)
       ref
     thor (0.20.0)
     thread_safe (0.3.6)
-    tilt (2.0.1)
-    tzinfo (1.2.3)
+    tilt (1.4.1)
```
Comment (edeyoung) [inline Gemfile.lock:232]: this also went down a version

Comment (ssayer) [reply Gemfile.lock:232]: This is expected. The various projects were on either Tilt 1.4 or 2.0. We decided to align on them all on Tilt 1.4.

**Review by ssayer — DISMISSED** [review]

File: Gemfile.lock (hunk 2 above)
Comment (ssayer) [reply Gemfile.lock:232]: I should add we chose 1.4 because most of the projects were on 1.4, I think only one or two were on tilt 2.0.

**Review by edeyoung — APPROVED** [review]

**Review by ssayer — APPROVED** [review]
